server/app/views/applicant/ProgramIndexView.java

```diff
@@ -192,6 +199,20 @@ private DivTag mainContent(
                 Math.max(relevantPrograms.unapplied().size(), relevantPrograms.submitted().size()),
                 relevantPrograms.inProgress().size()));
 
+    // Find all the categories that are on any of the relevant programs
+    List<String> relevantCategories =
+        relevantPrograms.allPrograms().stream()
```

Comment:
**question** Is there a specific sort order we want to apply, or do we not care?

Reply:
Good question.  I'll check with Product/UX and respond back here when I get an answer.

Reply:
They prefer that it be sorted alphabetically.  I added the sorting.  Thanks for noticing that!